e2-studio-star-rx72n-firmware/libs/rx_usb_comm/src/rx_usb_comm.c

```
@@ -1332,53 +1273,110 @@ rx_err_t rx_usb_comm_is_ready(const rx_usb_comm_handle_t* handle, bool* ready)
-/**
- * @brief Get current TX sequence number
- * @param[in] handle USB communication handle
- * @return Current TX sequence number (0 if handle is nullptr)
+/* =============================================================================
+ * Control Frame API
+ * =============================================================================
  */
-uint16_t rx_usb_comm_get_tx_sequence(const rx_usb_comm_handle_t* handle)
+
+rx_err_t rx_usb_comm_set_control_callbacks(rx_usb_comm_handle_t* handle,
+                                            void (*on_ping_cb)(const rx_frame_t* frame, void* ctx),
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_usb_comm_set_control_callbacks' of similar type ('void (*)(const int *, void *)') are easily swapped by mistake

(bugprone-easily-swappable-parameters)